guitar-dongle/src/main.cpp

```diff
@@ -0,0 +1,134 @@
+#include <Arduino.h>
+#include <WiFi.h>
+#include <unordered_map>
+#include <esp_now.h>
+#include "esp_wifi.h" 
+#include "midi_common.h"
+
+#define MIDI_UART_RX_PIN 16
+
+const unsigned long WIRELESS_TIMEOUT_MILLIS = 1500;               //If wee don't receive wireless (esp-now) ping from the device in 1.5s we note off currently on'ed notes.
+const unsigned long WIRELESS_IGNORE_AFTER_LAST_UART_WRITE = 3000; //We will ignore wireless (esp-now) messages if we received direct MIDI messages in last 3s
+
+enum Protocol {
+  UART,
```

Comment:
maybe `MIDI` would be a more clear protocol as `UART` is more used as a console input for debug output / commands